### DISCUSSION TITLE
Overall utility for making comics/manga/doujin

### DISCUSSION THREAD

**Discussion by Sorry-Im-Stupid** [Acly/krita-ai-diffusion#1410] [General]
I put this in General because it doesn't really fit any other category, but that may change over time.
When I found this I got excited because of the potential for generating sections of comic pages, to fulfill one of my long-standing ideas for a comic series.
I tried to do it in 2D ages ago but I just can't. I've tried to learn perspective in 2D many times over the years but my brain's just like 'Best I can do is different size stick figures, sorry bruh'.
And I tried to do it in 3D, but the program I tried to do it in wasn't made for that kind of format.
Now with this in Krita I may be able to finally make something. Set the layout and use controlnet for character positions then have the AI actually generate the lines and whatnot.
So I guess this discussion will be about that journey, and we can compare notes on such if anyone else is trying to do something similar.

**Comment by blazekicker** [comment]
i am trying the same thing right now and i am getting somewhere and no where at the same time

**Comment by Marksusu** [comment]
学习二维透视显然比学习使用 AI 工具更简单（对我来说 ：）
Learning perspective is obviously easier than learning to use AI tools ( to me :)
学习透视还可以让你能从 AI 生成的结果中快速选出最适用的方案。
Learning to pivot also allows you to quickly choose the best solution from the results of the AI output.

**Reply by blazekicker** [reply]
both are diffcuit for me i need to start getting somewhere with comics and if ai gets me there so be it

**Reply by Sorry-Im-Stupid** [reply]
My brain just doesn't do perspective in 2D.

**Comment by freecoderwaifu** [comment]
Make your comic layout either manually or in comic oriented software, either free or paid (Clip Studio EX). A1111/Forge has a comic layout extension, there might also be a Comfy UI workflow somewhere. Then just vaguely sketch each panel and use Scribble ControlNet panel by panel. Manually retouch AI quirks and do a refine pass or two. 

Or pose free 3D models and use Depth or any other ControlNet, or the Pose ControlNet, there are many ways.
